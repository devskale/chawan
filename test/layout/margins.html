<!DOCTYPE html>
<style>

* {
  font-family: monospace,monospace
}
</style>
<div style="display: inline-block; width: 15ch">
<div style="margin-bottom: 2ch;">wtf?</div>
<div style="display: flow-root; margin-top: 4ch">hi from right</div>
<div style="margin-left: 2ch; margin-right: 2ch; margin-top: 4ch">
<div style="display: flow-root">hi from left</div>
</div>
<div>
hi in center
</div>
</div>
