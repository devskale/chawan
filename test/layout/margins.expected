wtf?


hi from right


  hi from
  left
hi in center
